go/vt/mysqlctl/metadata_tables.go

```diff
@@ -18,12 +18,23 @@ const sqlCreateLocalMetadataTable = `CREATE TABLE IF NOT EXISTS _vt.local_metada
   PRIMARY KEY (name)
   ) ENGINE=InnoDB`
 
-// populateLocalMetadata creates and fills the _vt.local_metadata table,
-// which is a per-tablet table that is never replicated. This allows queries
+const sqlCreateShardMetadataTable = `CREATE TABLE IF NOT EXISTS _vt.shard_metadata (
```

Comment:
Please add a comment somewhere that this should be kept in sync with the test script that creates the tables. Or if not in sync, at least for handling backward compatibility.